map_model/src/objects/building.rs

```diff
@@ -227,110 +227,193 @@ fn sidewalk_to_bike(sidewalk_pos: Position, map: &Map) -> Option<(Position, Posi
 /// Businesses are categorized into one of these types.
 #[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
 pub enum AmenityType {
-    Groceries,
-    Food,
     Bar,
-    Medical,
-    Religious,
-    Education,
+    Beauty,
+    BikeStore,
+    BikeParkRent,
+    CarRepair,
+    CarShare,
+    Childcare,
+    ConvenienceStore,
+    Culture,
     Financial,
+    Food,
+    Laundry,
+    Medical,
+    Pet,
     PostOffice,
-    Culture,
-    Childcare,
+    Religious,
+    School,
     Shopping,
+    Supermarket,
+    University,
 }
 
 impl AmenityType {
     fn types(self) -> Vec<&'static str> {
-        // TODO: create categories for:
-        // hairdresser beauty chemist
-        // car_repair
-        // laundry
-
         match self {
-            AmenityType::Groceries => vec!["convenience", "supermarket"],
-            // TODO Sort
+            AmenityType::Bar => vec!["bar", "lounge", "pub", "nightclub", "biergarten"],
+            AmenityType::Beauty => vec!["hairdresser", "beauty", "chemist", "cosmetics"],
+            AmenityType::BikeParkRent => vec!["bicycle_parking", "bicycle_rental"],
```

Comment:
How do you envision we might use the BikeParkRent amenity?

It's not clear to me that someone would want one when looking for the other.

Reply:
When I looked at OSM in certain cities I noticed some bike shares (like citibike in NYC) were being tagged as bike parking. However, I now realize after @dabreegster comment below that these are only buildings being tagged in the 15min section, so maybe better to remove altogether.

Reply:
Ah, I see. Thanks for explaining.